instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import CommentsContainer from "../CommentSection/CommentSection";
+import "./PostContainer.css";
+import PropTypes from "prop-types";
+
+class PostContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.post.comments,
+      newComment: ""
+    };
+    console.log(props.post.comments);
+  }
+
+  changeHandler = event => {
+    console.log(" changehandler fired");
+    this.setState({ newComment: event.target.value });
+  };
+
+  addNewComment = event => {
+    console.log("add comment fired");
+    event.preventDefault();
+    const newComment = { username: "AdamHinkley", text: this.state.newComment };
+    const comments = this.state.comments.slice();
+    comments.push(newComment);
+    this.setState({ comments, newComment: "" });
+    console.log("add comment:", comments);
+  };
+
+  render() {
+    return (
+      <div className="post-container">
+        <div className="post-top-section">
+          <img className="thumbnail" src={this.props.post.thumbnailUrl} alt="thumbnail" />
```

Comment:
should be using `props.filteredposts` but that won't really work because the props.filteredposts is an array and this component is expecting an object.